.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -1,10 +1,12 @@
 ---
 name: Custom issue template
-about: Describe this issue template's purpose here.
+about: Issue template for custom issues.
 title: ''
 labels: ''
 assignees: ''
 
 ---
 
+***NOTE:*** *Please ask your "how to" questions or feature existence queries on the [stackoverflow](https://stackoverflow.com/questions/tagged/ant-media-server) or [our community forum](https://groups.google.com/forum/#!forum/ant-media-server).*
```

Comment:
I recommend something more basic like below

***Attention:*** For your support questions, ask at [Stackoverflow](https://stackoverflow.com/questions/tagged/ant-media-server) or [Google Group](https://groups.google.com/forum/#!forum/ant-media-server)

Reply:
changed